presto-docs/src/main/sphinx/release/release-347.md

```diff
@@ -0,0 +1,55 @@
+# Release 347 (XX Nov 2020)
+
+## General Changes
+
+* Add `ALTER VIEW ... SET AUTHORIZATION` syntax to allow changing the view owner. ({issue}`5789`)
+* Add support for `ALL` qualifier for `INTERSECT` and `EXCEPT`. ({issue}`2152`)
+* Add {func}`contains_sequence` function. ({issue}`5593`)
+* Add support for configuring subnet based network topology for topology-aware scheduling. ({issue}`4862`) 
+* Improve query performance by reducing worker to worker communication overhead. ({issue}`5905`, {issue}`5949`)
+* Improve result throughput over fast network links by making http response compression configurable. Clients
+  can disable compressed responses using the `--disable-compression` flag or `disableCompression` driver property.
+  Compression can be disabled server-wide with `query-results.compression-enabled=false`. ({issue}`5818`)
+* Rename ``rewrite-filtering-semi-join-to-inner-join`` session property to ``rewrite_filtering_semi_join_to_inner_join``. ({issue}`5954`)
+* Throw a user error for failing to decode a session property value. ({issue}`5731`)
+* Fix failure when expressions that produce values of type `row` are used in a `VALUES` clause. ({issue}`3398`)
```

Comment:
what does "fix failure" mean? Is it "Improve reported error message"?

Reply:
Oh - the phrasing is good. 
Optionally maybe `Fix query failure` instead `Fix failure`